modules/data.land/R/ens.veg.module.R

```diff
@@ -0,0 +1,53 @@
+.ens.veg.module <- function(getveg.id, dbparms, 
```

Comment:
Undocumented and un-exported function. Also, why are we starting these function names with a `.`?

Reply:
as these functions are entirely for within package use, I thought it was kind of a convention to do that, but no other particular reason behind. I can get rid of them

Reply:
More important (holding up the PR) is the lack of documentation and export

Reply:
I'll jump in to defend unexported as a good thing here -- If it's for in-package use, not exporting it means we can move/rename/delete it later without a deprecation rigamarole. And if we find later that we do need to export it, it's just as easy to do then as now.

(I do vote for a line or two of documentation, though)